standard/expressions.md

```diff
@@ -682,6 +687,7 @@ When a generic method is called without specifying type arguments, a ***type inf
 
 > *Example*: Given the method declaration:
 >
+> <!-- Example: {template:"standalone-lib", name:"TypeInference", expectedErrors:["x","x"], expectedWarnings:["x","x"]} -->
```

Comment:
Presumably this doesn't pass, so please give it an appropriate "problem" prefix, e.g. IncompleteExample.

Reply:
agreed